rllib/algorithms/tests/test_algorithm_config.py

```diff
@@ -25,7 +26,7 @@ def tearDownClass(cls):
     def test_running_specific_algo_with_generic_config(self):
         """Tests, whether some algo can be run with the generic AlgorithmConfig."""
         config = (
-            AlgorithmConfig(algo_class=PPO)
+            AlgorithmConfig(algo_class=DQN)
```

Comment:
Generic config does not work for PPO iff we enable RLModules by default.

Reply:
Same for other algorithms that need an exploration config.

Reply:
This breaks backward compatibility, is there a message that we can send here? some warning or error for misuse?